enterprise/frontend/src/embedding-sdk/store/selectors.ts

```diff
@@ -1,6 +1,11 @@
 import type { SdkStoreState } from "embedding-sdk/store/types";
 
-export const getIsLoggedIn = (state: SdkStoreState) => state.sdk.isLoggedIn;
+export const getLoginStatus = (state: SdkStoreState) => state.sdk.loginStatus;
+
 export const getIsInitialized = (state: SdkStoreState) =>
-  state.sdk.isInitialized;
+  state.sdk.loginStatus.status !== "uninitialized";
+
+export const getIsLoggedIn = (state: SdkStoreState) =>
+  getLoginStatus(state).status === "success";
```

Comment:
nit: I'd either inline `state.sdk.loginStatus` or use `getLoginStatus(state)` for both selectors. I'd prefer using `getLoginStatus` for readability though.

```suggestion
  getLoginStatus(state).status !== "uninitialized";

export const getIsLoggedIn = (state: SdkStoreState) =>
  getLoginStatus(state).status === "success";
```